docs/articles/documentation/index.md

```diff
@@ -6,7 +6,8 @@ inMainMenu: true
 ---
 # Documentation
 
-* [Article1](.)
-* [Article2](.)
-* [Article3](.)
-* [Article4](.)
\ No newline at end of file
+* [Getting Started](.)
+* [Test API](.)
+* [Using TestCafe](/testcafe/documentation/using-testcafe)
```

Comment:
Would be great! What if we file an issue and do this in a separate PR.
I'm already working on anchors in my `master`